.dockerignore

```diff
@@ -2,7 +2,7 @@
 .github
 .gitignore
 .gitlab-ci.yml
-**/target
+**/target*
```

Comment:
This was unintentional. But maybe it doesn't hurt. I have a separate rust-analyser target dir...